jenkins/ansible-lint.Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+pipeline {
+   agent any
+
+   stages {
+      stage('Clone') {
+         steps {
+            git 'https://github.com/progmaticltd/homebox.git'
+         }
+      }
+      stage('Installation playbooks') {
+         steps {
+            sh 'ansible-lint install/playbooks/main.yml'
+         }
+      }
+      stage('Testing playbooks') {
+         steps {
+            sh 'ansible-lint tests/playbooks/main.yml'
+         }
+      }
+      stage('Uninstall playbooks') {
+         steps {
+            sh 'ansible-lint uninstall/playbooks/*.ml'
```

Comment:
Looks like a typo:  `*.ml` → `*.yml`?

Reply:
Thanks, well spotted. I will fix it before merging